### DISCUSSION TITLE
第3节课作业(9班)

### DISCUSSION THREAD

**Discussion by SaigeRing** [InternLM/Tutorial#178] [作业]
## 基础作业：

复现课程知识库助手搭建过程 (截图)

## 进阶作业：

选择一个垂直领域，收集该领域的专业资料构建专业知识库，并搭建专业问答助手，并在 [OpenXLab](https://openxlab.org.cn/apps) 上成功部署（截图，并提供应用地址）

**Comment by xiaomile** [comment]
[第三节课作业](https://github.com/xiaomile/InternLM-homework/blob/main/%E4%BD%9C%E4%B8%9A2/readme.md)

**Comment by wangxuejiao** [comment]
[第3节课作业](https://zhuanlan.zhihu.com/p/677171567)

**Comment by ShelterWFF** [comment]
class3: http://t.csdnimg.cn/VcKcK

**Comment by KJ-Huang-sz** [comment]
https://blog.csdn.net/qq_40831032/article/details/135534898

**Comment by liuxianyi** [comment]
[📒标注的为作业](https://liuxianyi.notion.site/InternLM-langChain-dcc7a0d113494090a573734f805583e7?pvs=4)

**Comment by cookieswolf** [comment]
[03-InternLM  langchain 知识库](https://mubucm.com/doc/7gk2LGrGTGO)

**Comment by haizeigh** [comment]
基于internlm和langchain搭建知识库作业 https://note.youdao.com/s/dK1CnGhG

**Comment by xingyuan3** [comment]
作业：https://www.mubu.com/doc/4eA4152hAjd

**Comment by SaigeRing** [comment]
https://zhuanlan.zhihu.com/p/677700494

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135574909

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135579476?spm=1001.2014.3001.5501

**Comment by nini0919** [comment]
https://morning-gaura-eed.notion.site/514d1d5bcae7488db69d091efbd5b4bd?pvs=4

**Comment by nxznm** [comment]
https://zhuanlan.zhihu.com/p/677351372

**Comment by wocclyl** [comment]
https://m1ti1x9upsq.feishu.cn/wiki/EBqWwcGTsiqKnSkpHOBciDQhngf?from=from_copylink

**Comment by RangeKing** [comment]
https://rangeking.github.io/#/page/3.%20internlm%20with%20langchain
